masterlist.yaml

```diff
@@ -1449,6 +1453,9 @@ plugins:
       - link: 'https://www.nexusmods.com/oblivion/mods/21104/'
         name: 'Common Oblivion (Cobl)'
   - name: 'Cobl Main.esm'
+    url:
+      - link: 'https://www.nexusmods.com/oblivion/mods/21104/'
+        name: 'Common Oblivion (Cobl)'
```

Comment:
Not sure on that one; I think the existing regex is actually too hungry to begin with since it captures _any_ plugin that contains "Cobl" in the filename even if they are from different mod projects (e.g. my alchemy station and Sutch Coblized projects, but also pretty much any other pure Coblizer project). Most of the actual Cobl plugins all start with "Cobl " -- only some dev patches, _Salmo the Baker Cobl_, and _Item Interchange_ plugins in the Cobl download don't (and the last are already defined in their own section anyway). `Cobl.*\.es[mp]` might be a better regex to limit the link to just the actual Cobl files, possibly adding another item for `Salmo the Baker, Cobl.*\.esp` to catch those too. Alternatively, `(Salmo the Baker, )?Cobl.*\.es[mp]`?

Reply:
`(Salmo the Baker, )?Cobl.*\.es[mp]` is fine. Locations are primarily for maintainers, so, as long as plugins from the same page are all grouped together, plugins from separate pages are separated, & there's a location somewhere in each grouping, it should be clear which plugins come from where.